ros_buildfarm/common.py

```diff
@@ -280,6 +280,7 @@ def get_short_os_code_name(os_code_name):
     os_code_name_mappings = {
         'artful': 'A',
         'bionic': 'B',
+        'buster': 'DB',
```

Comment:
This should be just `B`. Similar to Saucy / Stretch the code name mapping alone isn't unique but it always appears the a character identifying for example Debian  / D.

Reply:
Just B in f22b00f